src/QA/QA.Web/ViewModels/ReviewViewModel.cs

```diff
@@ -108,6 +108,10 @@ public class
 
         public bool HasAutomatedQaResults => AutomatedQaResults.Any();
         public string PageTitle { get; set; }
+        public VacancyStatus VacancyStatus { get; set; }
+        public bool IsVacancyDeleted { get; set; }
+
+        public bool ShowDeletedMessage => IsVacancyDeleted && VacancyStatus == VacancyStatus.Referred;
```

Comment:
Should this care about vacancy status? I'd assume we would always want to show this message if the vacancy is deleted regardless of its status.

Reply:
i have removed this property now as i can directly use IsVacancyDeleted.